autoload/libexec/__fzf::widget::replace

```diff
@@ -0,0 +1,26 @@
+#!/bin/zsh
+
+local -a items
+IFS=$'\n' items=(`cat`)
+
+if [[ -n $items ]]; then
+  BUFFER=''
+  CURSOR=$#BUFFER
+  zle redisplay
+  for item in $items; do
+    if [[ $1 = -q ]]; then
+      # quote special characters with backslashes
+      BUFFER+="$item:q:gs/\\~/~/ "
+    else
+      BUFFER+="$item "
```

Comment:
@ytet5uy4, one more small issue: when I paste command from history, I got one or two spaces at the end of command, see gif:
![pjbvr86bll](https://user-images.githubusercontent.com/3027126/33028483-e60958be-ce37-11e7-84ec-0367f1f7c0e2.gif)

Maybe would be better to remove space at the end on or remove it before command execution?

When I insert command from history, usually I don't want to append something to the end.

I just remove this space from this line and it solve issue for me.

Reply:
@popstas
Thank you for report!
I fixed this bug at https://github.com/ytet5uy4/fzf-widgets/pull/34.

Reply:
Thanks!